.github/stale.yml

```diff
@@ -1,11 +1,11 @@
 # Configuration for probot-stale - https://github.com/probot/stale
 
 # Number of days of inactivity before an Issue or Pull Request becomes stale
-daysUntilStale: 180
+daysUntilStale: 30
```

Comment:
this seems really aggressive.

Reply:
maybe we talk offline

Reply:
How does 60 sound?

Reply:
Just to put everyone's mind at ease (I hope), as part of our weekly triage, we will also be looking at issues that are about to go stale and can reset the clock where needed. The stalebot will not be working unchecked. The goal is to fix all the issues we've tagged these last two months and if that takes 4-6 moths...so be it.